README.md

```diff
@@ -7,6 +7,8 @@ This project very much under active development, and is currently not as reusabl
 
 ## Configuring for Development
 
+Make sure an instance of elasticsearch is available at `http://elasticsearch.local:9200/`. If you are running theonion/onion-services you can add `elasticsearch.local` to your `/etc/hosts` line that points at that docker dev machine.
```

Comment:
I'm going to commit an alternative approach today, which avoids messing with /etc/hosts. Instead just set ELASTICSEARCH_HOST environment variable. 

https://github.com/theonion/django-bulbs/pull/129/files#diff-0083a71d12fcbc15cd2f91d55f4cf579R87